### PR TITLE
Sub pages

### PR DESCRIPTION
**Pull request by jdonisvitch**
Update pages with markdown
Allow anchors to work with github markdown
Simplify sub pages, make not so ugly
Add primer-markdown
Add basic styles for markdown

### CODE REVIEW

File: lib/public/js/main.js [hunk 1]
```diff
@@ -61,5 +66,27 @@ const newsletterSubmit = () => {
     };
 };
 
+const scrollToElement = (selector) => {
```
Comment (devinivy) [inline lib/public/js/main.js]: This is a pretty sweet little solution for now, but would love to remove it once we figure-out whatsup!

File: package.json [hunk 2]
```diff
@@ -28,10 +28,12 @@
     "handlebars": "4.x.x",
     "hapi": "16.x.x",
     "haute-couture": ">=2.1.0 <3",
+    "highlight.js": "^9.12.0",
```
Comment (devinivy) [inline package.json]: Can you remove this if we're not going to use it?

File: lib/public/css/markdown.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
+.markdown-body {
```
Comment (devinivy) [inline lib/public/css/markdown.scss]: Just gotta make sure code blocks get a monospaced font.

File: package.json (hunk 2 above)
Comment (jdonisvitch) [reply package.json]: Oops! Totally left that by accident, cleaning up!

**Pull request merged**